app/assets/javascripts/discourse/app/controllers/composer.js

```diff
@@ -625,7 +626,7 @@ export default Controller.extend({
 
   disableSubmit: or("model.loading", "isUploading"),
 
-  save(force) {
+  save(force, options) {
```

Comment:
maybe we just go with `save(force, options = {})` so we don't have to use optional chaining